mlrun/api/api/endpoints/model_endpoints.py

```diff
@@ -48,7 +49,7 @@ def create_or_patch(
         )
     ModelEndpoints.create_or_patch(
         db_session=db_session,
-        access_key=access_key,
+        access_key=os.environ.get("V3IO_ACCESS_KEY"),
```

Comment:
Since we are moving to a automatic initialization of the model endpoint records. and the records are now being written to pipelines` name space, we have to use pipelines` credentials as well. If you see any other option regarding this, let me know...

Reply:
Great, add that in a comment in the code